tests/test_sftp_automatic.py

```diff
@@ -0,0 +1,123 @@
+from base import TestSFTPBase
+import tap_tester.connections as connections
+import tap_tester.menagerie   as menagerie
+import tap_tester.runner      as runner
+import os
+import csv
+import json
+
+class TestSFTPAutomatic(TestSFTPBase):
+
+    def name(self):
+        return "tap_tester_sftp_automatic_fields"
+
+    def get_files(self):
+        return [
+            {
+                "headers": ['id', 'string_col', 'integer_col'],
+                "directory": "table_1_files",
+                "files": ["table_1_fileA.csv"],
+                "num_rows": 50,
+                "generator": self.generate_simple_csv_lines_typeA
+            }
+        ]
+
+    def setUp(self):
```

Comment:
Add function comments for each file's function.

Reply:
Added function comments